TrafficCapture/trafficCaptureProxyServer/src/main/resources/log4j2.properties

```diff
@@ -8,4 +8,18 @@ appender.console.layout.pattern = [%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c
 
 rootLogger.level = info
 rootLogger.appenderRefs = stderr
-rootLogger.appenderRef.stderr.ref = STDERR
\ No newline at end of file
+rootLogger.appenderRef.stderr.ref = STDERR
+
+appender.metrics.type = Console
+appender.metrics.name = METRICS
+appender.metrics.layout.type = JsonLayout
```

Comment:
Please don't set this to JsonLayout by default?  I'd rather have a user turn this on to work with their metrics consumers rather than relegate humans to be second-class citizens of their logs.